README.md

```diff
@@ -302,82 +218,80 @@ As the Stellar network grows, the Stellar Core instance has to catch up on an in
 ```
 
-This command exports takes in a start and end time and converts it to a ledger range. The ledger range that is returned will be the smallest possible ledger range that completely covers the provided time period. 
+This command takes in a start and end time and converts it to a ledger range. The ledger range that is returned will be the smallest possible ledger range that completely covers the provided time period.
 
 <br>
-<br>
+
+---
 
 # Schemas
 
 See https://github.com/stellar/stellar-etl/blob/master/internal/transform/schema.go for the schemas of the data structures that are outputted by the ETL.
 
-<br>
 <br>
 
+---
+
 # Extensions
+
 This section covers some possible extensions or further work that can be done.
 
 ## **Adding New Commands**
+
 In general, in order to add new commands, you need to add these files:
 
- - `export_new_data_structure.go` in the `cmd` folder
-	 - This file can be generated with cobra by calling: `cobra add {command}`
-	 - This file will parse flags, create output files, get the transformed data from the input package, and then export the data.
- - `export_new_data_structure_test.go` in the `cmd` folder
-	 - This file will contain some tests for the newly added command. The `runCLI` function does most of the heavy lifting. All the tests need is the command arguments to test and the desired output.
-	 - Test data should be stored in the `testdata/new_data_structure` folder
- - `new_data_structure.go` in the `internal/input` folder
-	 - This file will contain the methods needed to extract the new data structure from wherever it is located. This may be the history archives, the bucket list, or a captive core instance. 
-	 - This file should extract the data and transform it, and return the transformed data.
-	 - If working with captive core, the methods need to work in the background. There should be methods that export batches of data and send them to a channel. There should be other methods that read from the channel and transform the data so it can be exported.
+- `export_new_data_structure.go` in the `cmd` folder
+  - This file can be generated with cobra by calling: `cobra add {command}`
+  - This file will parse flags, create output files, get the transformed data from the input package, and then export the data.
+- `export_new_data_structure_test.go` in the `cmd` folder
+  - This file will contain some tests for the newly added command. The `runCLI` function does most of the heavy lifting. All the tests need is the command arguments to test and the desired output.
+  - Test data should be stored in the `testdata/new_data_structure` folder
+- `new_data_structure.go` in the `internal/input` folder
+  - This file will contain the methods needed to extract the new data structure from wherever it is located. This may be the history archives, the bucket list, or a captive core instance.
+  - This file should extract the data and transform it, and return the transformed data.
```

Comment:
we're not actually transforming data in `internal/input`. This should move under `internal/transform`

Reply:
I think technically ledgers and assets does do a small transformation (LCM to the history archive struct for changes). But you're right we shouldn't be doing this and this should only be extracting data and the transforms should happen in the transform dir